skeleton/src/main/java/com/decathlon/vitamin/compose/skeleton/VitaminSkeleton.kt

```diff
@@ -0,0 +1,89 @@
+package com.decathlon.vitamin.compose.skeleton
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.decathlon.vitamin.compose.foundation.VitaminTheme
+import com.google.accompanist.placeholder.PlaceholderHighlight
+import com.google.accompanist.placeholder.placeholder
+
+object VitaminSkeleton {
+
+    /**
+     * Rectangular skeleton shape
+     * @param modifier The [Modifier] to be applied to this skeleton shape
+     * @param color color of this skeleton shape
+     * @param highlightColor color of the shimmer effect
+     * @param cornerRadius size of the rectangle shape corner radius, no radius by default
+     * @param contentDescription the [String] for the content description
+     */
+    @Composable
+    fun Rectangular(
+        modifier: Modifier = Modifier,
+        color: Color = VitaminTheme.colors.vtmnBackgroundTertiary,
+        highlightColor: Color = VitaminTheme.colors.vtmnBackgroundPrimary,
```

Comment:
Same here (and for all components in this object), we should probably used a class for all colors.